### PR TITLE
Add rule for WebLogic virtual-directory

### PR DESCRIPTION
**Pull request by mbriskar**

### CODE REVIEW

**Comment by lincolnthree** [comment]
What do you need help with?

**Comment by lincolnthree** [comment]
Please talk to me about this tomorrow.

**Comment by lincolnthree** [comment]
I think this should be put directly into rules-reviewed, with proper metadata, don't you?

**Comment by mbriskar** [comment]
sure, sorry help-wanted is deprecated

File: rules-reviewed/jboss-eap/weblogic/tests/XmlWebLogicRulesTest.windup.test.xml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0"?>
```
Comment (lincolnthree) [inline rules-reviewed/jboss-eap/weblogic/tests/XmlWebLogicRulesTest.windup.test.xml]: I think this file should be renamed to match the naming conventions of the rulesets:

xml-weblogic.windup.test.xml

wdyt?

File: rules-reviewed/jboss-eap/weblogic/xml-weblogic.windup.xml [hunk 2]
```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0"?>
+<ruleset xmlns="http://windup.jboss.org/v1/xml" id="weblogic_rules_xml">
+   <metadata>
+      <description>
+         This ruleset provides analysis of WebLogic proprietary xmls and constructs that may require individual attention when migrating to JBoss EAP 6+.
+      </description>
+      <dependencies>
+         <addon id="org.jboss.windup.rules,windup-rules-javaee,2.0.1.Final"/>			
+         <addon id="org.jboss.windup.rules,windup-rules-java,2.0.0.Final"/>
+	 <addon id="org.jboss.windup.rules,windup-rules-xml,2.0.0.Final"/>
```
Comment (jsight) [inline rules-reviewed/jboss-eap/weblogic/xml-weblogic.windup.xml]: I think that these should all use the same version numbers.

Comment (lincolnthree) [inline rules-reviewed/jboss-eap/weblogic/xml-weblogic.windup.xml]: +1

Comment (mbriskar) [inline rules-reviewed/jboss-eap/weblogic/xml-weblogic.windup.xml]: I copied it from somewhere

**Comment by lincolnthree** [comment]
Updated to pass validation and merged.

**Pull request closed without merging**